README.md

```diff
@@ -30,12 +30,23 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+The .forEach method changes the original array where .map will create a different array with an changes.
+
 2. What is the difference between a function and a method?
 
+A function describes the actions to be performed when invoked. A method is a built-in function to JavaSript that executes when invoked without writing out the actions to be performed.
```

Comment:
A method is really just a function inside of an object, if you create an object `const obj = {name: "Shawn", age: function(){
   // some function
}}` the age function is now a method on `obj`